doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -328,6 +328,16 @@ PSI Variables by Alpha
    Conventions for root indexing and whether h refers to transition or root
    irrep are as in :ref:`sec:psivarnotes`.
 
+.. psivar:: CCname ROOT n (h) -> ROOT m (i) EINSTEIN A (LEN)
+
+   The Einstein A coefficient, the spontaneous emission 'probability.'
+   Units are in [1/s].
+
+.. psivar:: CCname ROOT n (h) -> ROOT m (i) EINSTEIN B (LEN)
+
+   The Einstein B coefficient, the stimulated emission 'probability'
+   in terms of energy density. Units are in [m^3 / J / s^2].
```

Comment:
I periodically forget this, but we now try to save qcvars in au for compatibility with QCSchema and to avoid unit conversion error. I should plan on a pass (before v1.6) to switch to au. But that'll involve tests to check pint can to the au -> SI conversion easily. For now, it's great to get these saved and in at least one set of known units :-)